signature/sui_verify.go

```
@@ -0,0 +1,45 @@
+package signature
+
+import (
+	"encoding/base64"
+	"fmt"
+
+	suimodels "github.com/block-vision/sui-go-sdk/models"
+)
+
+func SuiVerifyPersonalMessage(message string, b64Signature string) (string, bool, error) {
```

empty message is invalid right?  if so, we should return an error if message is empty string

---

actually there are no restrictions on message length, also it will be prepended anyway with personal sign prefix, so it should be ok.